crates/nu-cli/src/app.rs

```diff
@@ -32,6 +44,14 @@ impl App {
     }
 
     pub fn main(&self, argv: &[String]) -> Result<(), ShellError> {
+        if self.perf() {
```

Comment:
Since we're starting the stopwatch here and only using stopwatch in main, you don't need the `lazy_static` dependency or initialization. I think you can just create a stopwatch here and start it going, and keep it as a normal variable

Reply:
The stopwatch is not only used in main. It's used in cli.rs as well. The plan was to flush out some other places to put stopwatch timings in the future as well.

Reply:
Got it, I missed the other file